### ISSUE TITLE
Outreachy Documentation Project: <Anshu_Kumari>

### ISSUE THREAD

**Issue by AnshuKumari197** [ersilia-os/ersilia#151]
Applicant: @AnshuKumari197
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [x] Create a README file with the name <README_yourname.md> under the /documentation folder
- [x] Link the #PR in a comment under this issue
- [x] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [x] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [ ] Incorporate feedback from the mentor
- [ ] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [ ] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [x] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [x] Test one model
- [x] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [x] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [x] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [x] Create one image / slide to explain Ersilia’s mission and vision
- [x] Link to the image/slide as a comment under this issue
- [x] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [ ] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [ ] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [ ] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [ ] Look up two other projects and comment on their issues with feedback on one of their tasks
- [x] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [ ] Add a comment in this issue with your new idea, tagging the mentor 
- [ ] Get feedback from the mentor and act accordingly
- [ ] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by AnshuKumari197** [comment]
hello @GemmaTuron here is the link to my previous PR that was merged by you. Please review and recommend.
#1
#84

**Comment by GemmaTuron** [comment]
Hi @AnshuKumari197 

Thanks for your contributions, you created quite a good README File, but there are some suggestions for your consideration:
- The table of Content could have links to each section to make it easier to follow
- The Project description has this sentence:  Ersilia Open Source Initiative is a part of the Ersilia Model Hub which should be the other way around, the Ersilia Open Source Initiative is the organization, and the Ersilia Model Hub is one of our services, products, offers.. can you rephrase the first paragraph? the second paragraph is already in the credit section, so you do not need to mention it here.
- Overview: I very much like that schema, but I think it is perhaps too detailed for a README file
- Features: please revise the license
- Operations: I think you could eliminate this section and merge the content with the project description, as is very much related to the Ersilia Model Hub?
- License: please double check the License file in the repo
- Requirements: the Note is referring to the models inside the hub, not the packages we need to run the hub. Some of the requirements are not needed, like streamlit. Can you revise this list?
- The vision is part of the Ersilia Open Source Initiative, so perhaps it would be better placed with the About Us?
Future Scope and Attributes: The Readme file is referring to the Ersilia Model Hub, not the outreachy project specifically, so the vision etch should refer to the Hub not documentation
- the Credit should mention how to cite our software, check the Citation File Format for more info on that

Let us know what you think of the aspects I mention above.

**Comment by Dapper50** [comment]
Hi @AnshuKumari197 Good job, please note that the community section is required for you to make a final application,
I can see that you have checked the final application box. Kindly make an edit to the community section.

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron i have updated my readme file under #190 according to you suggestions please check it once and give feedback

**Comment by GemmaTuron** [comment]
Hi @AnshuKumari197 !

Thanks for the changes I merged them! go ahead with your next steps!

**Comment by AnshuKumari197** [comment]
Hi @GemmaTuron , 
there was an issue left in my readme file I edited it . Issue - The links present in table of content is opening the editing file instead of the real readme file.
Please merge it once again and sorry for you inconvenience.

**Comment by AnshuKumari197** [comment]
Hii @GemmaTuron ,
I have created a slide based on Ersilia Model Hub mission and vision.
![1](https://user-images.githubusercontent.com/84778340/162557233-e5a1f11a-2fa6-4290-9137-2efef4ade05e.jpg)

![2](https://user-images.githubusercontent.com/84778340/162557290-5ff99654-16d8-482f-abc0-dffe31b0c404.jpg)
![3](https://user-images.githubusercontent.com/84778340/162557298-a1207779-33a1-46bc-a7b3-36469812d14b.jpg)
![4](https://user-images.githubusercontent.com/84778340/162557300-cdae8a2a-7f60-4d7a-92d7-0aa61b95877d.jpg)

**Comment by GemmaTuron** [comment]
Hi @AnshuKumari197 

Thanks for the work! A few comments on the slides:
- the design is cool, but can you use Ersilia's colours? (plum and green for example)
- Like in the README file, it is confusing to talk about the Ersilia Model Hub before the mission and vision. Ersilia, as an organisation, has a mission and vision, and to fulfill the mission and vision it has created the Ersilia Model Hub. I would invert the order
- Mission and vision statements: they are quite focused on the Outreachy program, but in this case we refer to the Ersilia's organization vision and mission, you can check the strategic plan to know more about it.

**Comment by GemmaTuron** [comment]
while preparing the final application on the outreachy website, please link this PR there

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron I updated the slides according to your feedback. Please check once again and review it.
![Ersilia (1)](https://user-images.githubusercontent.com/84778340/163254525-9b1bb0a4-aa75-44da-b283-bed938b15681.png)
![Ersilia (2)](https://user-images.githubusercontent.com/84778340/163254579-3dad3aa0-94da-4031-a980-15a1e8999667.png)
![Ersilia (3)](https://user-images.githubusercontent.com/84778340/163254605-dc86bff3-837f-4b00-a5d5-8fc38f09200a.png)
![Ersilia (4)](https://user-images.githubusercontent.com/84778340/163254643-97f64a6e-b896-4776-a65e-1e425688fc74.png)
![3](https://user-images.githubusercontent.com/84778340/163254932-ca937c1d-05d6-4173-9292-a576f4485f52.png)
![4](https://user-images.githubusercontent.com/84778340/163255147-f301cb3a-14d8-4a06-9f25-8b84fe4bb6f4.png)
![5](https://user-images.githubusercontent.com/84778340/163255227-0b5d3fd9-e8e3-44c7-a120-671dd9ee7182.png)
![6](https://user-images.githubusercontent.com/84778340/163255276-418ba586-c541-4f5b-8039-105e11b69cae.png)
![7](https://user-images.githubusercontent.com/84778340/163255334-86059555-78aa-4251-93c3-883eb9941a6c.png)

**Comment by GemmaTuron** [comment]
Lot of effort @AnshuKumari197 really appreciate! 
Thanks!

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron Thankyou so much for your support and guidance .

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron I install Ersilia Model Hub on my PC.
![image](https://user-images.githubusercontent.com/84778340/163671269-a25efe20-3ff6-40eb-8660-9aded6406e47.png)

**Comment by GemmaTuron** [comment]
Hi @AnshuKumari197 that's fantastic!
Did you find problems in the installation?

**Comment by AnshuKumari197** [comment]
Yes, I faced some issues.
Some problem in installation of Ubuntu.
Issues in running Ersilia Model hub.
And I was doing one silly mistake that was " pip install -e . "  I missed this last dot while installing which created lot of issues.

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron I know I am late to make a blog post on AI/ML in biomedical. But I made it. Please check once and give some your valuable feedback. 
Thank you!
https://docs.google.com/document/d/1NJudyKArILgTgNC-iON8jpUvJGZsvjwCZsy1Ky4MY0o/edit?usp=sharing

**Comment by AnshuKumari197** [comment]
Hi, @GemmaTuron here is my strategic plan 2021-2023.
https://docs.google.com/document/d/1eKO4ZM-PyK5usGxwDGC_J7N4h4ui5IsdgYaiaOLQE7E/edit?usp=sharing
